pyexchange/uniswapv2_analytics.py

```diff
@@ -266,24 +305,39 @@ def get_trades(self, pair: str, page_number: int = 1) -> List[Trade]:
         pair_address = self.get_pair_address(base_token.address, quote_token.address)
 
         mint_events = self.get_our_mint_txs(pair_address)
+        burn_events = self.get_our_burn_txs(pair_address)
         last_mint_event = mint_events[0]
+        last_burn_event = mint_events[0]
 
         our_liquidity_balance = Wad.from_number(last_mint_event['liquidity'])
         current_liquidity = self.get_current_liquidity(pair_address)
 
         last_mint_timestamp = int(last_mint_event['timestamp'])
+        last_burn_timestamp = int(last_burn_event['timestamp'])
         two_days_ago_unix = int(time.time() - (49 * 60 * 60))
+        mints_in_last_two_days = list(filter(lambda mint: int(mint['timestamp']) > two_days_ago_unix, mint_events))
```

Comment:
At some point in the future, it'll make sense to filter time in the query rather than querying years of history.  Not sure if _thegraph_ supports such a filter.

Reply:
Yeah that's a good point - I had looked into it but, unfortunately, they don't really have a great time based filter available. The closest approximation would be block height - but that would increase complexity correspondingly, and I figured it was easier to just do it on the client given this should only be called once an hour or so.

Reply:
Yeah that makes sense. We would be able to add logic around a starting timestamp / block, but I figured it wasn't worth the added complexity since we would only be making the call once an hour